Form/Handler/AuthorizeFormHandler.php

```diff
@@ -94,7 +94,7 @@ public function process()
         }
 
         $this->form->handleRequest($request);
-        if (!$this->form->isValid()) {
+        if ($this->form->isSubmitted() && $this->form->isValid() === false) {
```

Comment:
Why don't `!$this->form->isValid()`? Or is it from cs-fixer profile?

Reply:
I don't remember changing this line. Most likely it was done by cs-fixer.